old-ui/app/css/rif.css

```diff
@@ -1,3 +1,41 @@
+.confirmation-message {
```

Comment:
This file should be excluded from the repo since it's generated by the gulp task, i think it's in the .gitignore in master

Reply:
Yep, i have the exclusion right in the .gitignore...rare